.config/s3.config.php

```diff
@@ -12,6 +12,7 @@
         'region' => getenv('OBJECTSTORE_S3_REGION') ?: '',
         'hostname' => getenv('OBJECTSTORE_S3_HOST') ?: '',
         'port' => getenv('OBJECTSTORE_S3_PORT') ?: '',
+        'storageClass' => getenv('OBJECTSTORE_S3_STORAGE_CLASS') ?: 'STANDARD',
```

Comment:
Why do we need to decouple storageClass? Already defaults to standard [upstream](https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/primary_storage.html#simple-storage-service-s3):

> storageClass defaults to STANDARD